components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -6128,7 +6128,7 @@ def __init__(self, dictionary, descriptorProvider):
                                          descriptorProvider,
                                          isMember="Dictionary",
                                          defaultValue=member.defaultValue,
-                                         exceptionCode="return Err(());"))
+                                         exceptionCode="return Err(());\n"))
```

Comment:
Could you mention in the commit message the reason for this change? You mentioned it on IRC, but it may be useful for posterity having it there.

Reply:
Sure! Let me separate this diff into a single commit and describe more detail in the commit message. Thanks!

Reply:
@emilio Updated in bfbe7ebff ! If the commit message is not clear, please tell me! Thanks 😁